### ISSUE TITLE
Error reading data URL

### ISSUE THREAD

**Issue by meNESS** [danikula/AndroidVideoCache#60]
Hi,

Sometimes following exception will be thrown.

**Note:** I'm using the latest version of the AndroidVideoCache.

```
ProxyCache error
com.danikula.videocache.ProxyCacheException: Error reading data from http://collectup.blob.core.windows.net/videos/fc16f22a-e458-4056-8218-1d17ddbadb37.mp4
    at com.danikula.videocache.HttpUrlSource.read(HttpUrlSource.java:99)
    at com.danikula.videocache.ProxyCache.readSource(ProxyCache.java:126)
    at com.danikula.videocache.ProxyCache.access$100(ProxyCache.java:19)
    at com.danikula.videocache.ProxyCache$SourceReaderRunnable.run(ProxyCache.java:179)
    at java.lang.Thread.run(Thread.java:856)
 Caused by: java.net.SocketException: Socket is closed
    at java.net.PlainSocketImpl.checkNotClosed(PlainSocketImpl.java:134)
    at java.net.PlainSocketImpl.available(PlainSocketImpl.java:140)
    at java.net.PlainSocketImpl$PlainSocketInputStream.available(PlainSocketImpl.java:228)
    at java.io.BufferedInputStream.read(BufferedInputStream.java:328)
    at libcore.net.http.FixedLengthInputStream.read(FixedLengthInputStream.java:45)
    at java.io.BufferedInputStream.read(BufferedInputStream.java:304)
    at com.danikula.videocache.HttpUrlSource.read(HttpUrlSource.java:95)
    at com.danikula.videocache.ProxyCache.readSource(ProxyCache.java:126) 
    at com.danikula.videocache.ProxyCache.access$100(ProxyCache.java:19) 
    at com.danikula.videocache.ProxyCache$SourceReaderRunnable.run(ProxyCache.java:179) 
    at java.lang.Thread.run(Thread.java:856) 
```

**Comment by josemllz** [comment]
I have the same problem as you, is there any solution to solve this?

**Comment by aliselcuk** [comment]
Same here, E/MediaPlayer: error (1, -1007)

Works fine with the original url but not with the cached one.

**Comment by shaopx** [comment]
try this 
https://github.com/danikula/AndroidVideoCache/pull/70

**Comment by danikula** [comment]
Fixed in 2.5.0
